README.rst

```diff
@@ -30,13 +30,220 @@
 NVIDIA TensorRT Inference Server
 ================================
 
-    **NOTE: You are currently on the r19.09 branch which tracks
-    stabilization towards the next release. This branch is not usable
-    during stabilization.**
-
 .. overview-begin-marker-do-not-remove
 
+The NVIDIA TensorRT Inference Server provides a cloud inferencing
+solution optimized for NVIDIA GPUs. The server provides an inference
+service via an HTTP or gRPC endpoint, allowing remote clients to
+request inferencing for any model being managed by the server.
+
+What's New In 1.6.0
+-------------------
+
+* Added TensorRT 6 support, which includes support for TensorRT dynamic 
+  shapes.
+
+* Shared memory support is added as an alpha feature in this release. This 
+  support allows input and output tensors to be communicated via shared 
+  memory instead of over the network. Currently only system (CPU) shared 
+  memory is supported.
+
+* Amazon S3 is now supported as a remote file system for model repositories. 
+  Use the s3:// prefix on model repository paths to reference S3 locations.
+
+* The inference server library API is available as a beta in this release. 
+  The library API allows you to link against libtrtserver.so so that you can 
+  include all the inference server functionality directly in your application.
+
+* GRPC endpoint performance improvement. The inference server’s GRPC endpoint
+  now uses significantly less memory while delivering higher performance.
+
+* The ensemble scheduler is now more flexible in allowing batching and 
+  non-batching models to be composed together in and ensemble.
```

Comment:
"in an ensemble"

Reply:
Corrected the phrasing in the Framework release notes as well.